python/huggingfaceserver/huggingfaceserver/vllm/utils.py

```diff
@@ -37,15 +36,15 @@ def vllm_available() -> bool:
 
 def infer_vllm_supported_from_model_architecture(
     model_config_path: Union[Path, str],
-) -> Optional[Type[nn.Module]]:
+) -> bool:
     if not _vllm:
-        return None
+        return False
     model_config = AutoConfig.from_pretrained(model_config_path)
     architecture = model_config.architectures[0]
-    model_cls = ModelRegistry.load_model_cls(architecture)
-    if model_cls is None:
+
+    if architecture not in ModelRegistry.get_supported_archs():
         logger.info("not a supported model by vLLM")
```

Comment:
shouldn't it return false if there is supported archs?

Reply:
missed it, Thanks